components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/internal/http/api/ConfigurationLoader.java

```diff
@@ -107,54 +161,61 @@ private static InternalAPI createAPI(String classFQName) {
         if (obj instanceof InternalAPI) {
             return (InternalAPI) obj;
         } else {
-            handleException("Error creating Internal InternalAPI. The InternalAPI should be of type " +
-                    "InternalAPI");
+            handleException("Error creating Internal InternalAPI. The InternalAPI should be of type " + "InternalAPI");
         }
         return null;
     }
 
-
-    public static int getInternalInboundPort() {
+    public static int getInternalInboundHttpPort() {
 
         File synapseProperties = Paths.get(CarbonUtils.getCarbonConfigDirPath(), "synapse.properties").toFile();
         Properties properties = new Properties();
-        InputStream inputStream = null;
-        try {
-            inputStream = new FileInputStream(synapseProperties);
+
+        try (InputStream inputStream = new FileInputStream(synapseProperties)) {
             properties.load(inputStream);
         } catch (FileNotFoundException e) {
             handleException("synapse.properties file not found", e);
         } catch (IOException e) {
             handleException("Error while reading synapse.properties file", e);
-        } finally {
-            if (inputStream != null) {
-                try {
-                    inputStream.close();
-                } catch (IOException ignored) {
-                }
-            }
         }
-        int internalInboundPort = Constants.DEFAULT_INTERNAL_HTTP_API_PORT;
 
         String internalInboundEnabledProperty = properties.getProperty(Constants.INTERNAL_HTTP_API_ENABLED);
         if (internalInboundEnabledProperty == null) {
             return -1;
         }
+
         boolean internalInboundEnabled = Boolean.parseBoolean(internalInboundEnabledProperty);
         if (!internalInboundEnabled) {
             return -1;
         }
 
-        String internalInboundPortProperty =
-                properties.getProperty(Constants.INTERNAL_HTTP_API_PORT);
+        int internalInboundHttpPort = Constants.DEFAULT_INTERNAL_HTTP_API_PORT;
+        String internalInboundPortProperty = properties.getProperty(Constants.INTERNAL_HTTP_API_PORT);
+
         if (internalInboundPortProperty != null) {
             try {
-                internalInboundPort = Integer.parseInt(internalInboundPortProperty);
+                internalInboundHttpPort = Integer.parseInt(internalInboundPortProperty);
             } catch (Exception ex) {
                 handleException(Constants.INTERNAL_HTTP_API_PORT + " is not in proper format", ex);
             }
         }
-        return internalInboundPort;
+
+        internalInboundHttpsPort = Constants.DEFAULT_INTERNAL_HTTPS_API_PORT;
+        String internalInboundHttpsProperty = properties.getProperty(Constants.INTERNAL_HTTPS_API_PORT);
```

Comment:
internalInboundPortProperty vs internalInboundHttpsProperty? I think it needs to change